### ISSUE TITLE
WCAG 2.0 - Lista accessibilità A

### ISSUE THREAD

**Issue by Polpetta** [Hexamini/PassioneKaraoke#43]
Questa issue è il punto di riferimento per valutare tutti i vari punti dell'accessibilità per arrivare a ottenere il WCAG _A_. La checklist è suddivisa per livelli. Ogni punto completato va spuntato, in modo da tenere traccia di dove siamo arrivati col lavoro.
I commit che si riferiscono all'accessiblità di livello _A_ andranno segnati con il numero di questa issue, in modo da tenerne traccia e poterli riepilogare successivamente.
Le issue che si riferiscono a problemi che riguardano la checklist qui presenti dovranno citare questa issue, in modo di poterne tenere adeguatamente traccia
# **Checklist WCAG livello A**
- Text Alternatives ([Guildeline 1.1](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#text-equiv))
  - [x]  [Non-text Content](http://www.w3.org/TR/WCAG20/#text-equiv-all)
- Time-based Media ([Guideline 1.2](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#media-equiv))
  - [x] [Audio-only and Video-only (Prerecorded)](http://www.w3.org/TR/WCAG20/#media-equiv-av-only-alt)
  - [x] [Captions (Prerecorded)](http://www.w3.org/TR/WCAG20/#media-equiv-captions)
  - [x] [Audio Description or Media Alternative (Prerecorded)](http://www.w3.org/TR/WCAG20/#media-equiv-audio-desc)
- Adaptable ([Guideline 1.3](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#content-structure-separation))
  - [x] [Info and Relationships](http://www.w3.org/TR/WCAG20/#content-structure-separation-programmatic)
  - [x] [Meaningful Sequence](http://www.w3.org/TR/WCAG20/#content-structure-separation-sequence)
  - [x] [Sensory Characteristics](http://www.w3.org/TR/WCAG20/#content-structure-separation-understanding)
- Distinguishable ([Guideline 1.4](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast))
  - [x] [Use of Color](http://www.w3.org/TR/WCAG20/#visual-audio-contrast-without-color)
  - [x] [Audio Control](http://www.w3.org/TR/WCAG20/#visual-audio-contrast-dis-audio)
- Keyboard Accessible ([Guideline 2.1](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#keyboard-operation))
  - [x] [Keyboard](http://www.w3.org/TR/WCAG20/#keyboard-operation-keyboard-operable)
  - [x] [No Keyboard Trap](http://www.w3.org/TR/WCAG20/#keyboard-operation-trapping)
- Enough Time ([Guideline 2.2](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#time-limits))
  - [x] [Timing Adjustable](http://www.w3.org/TR/WCAG20/#time-limits-required-behaviors)
  - [x] [Pause, Stop, Hide](http://www.w3.org/TR/WCAG20/#time-limits-pause)
- Seizures ([Guideline 2.3](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#seizure))
  - [x] [Three Flashes or Below Threshold](http://www.w3.org/TR/WCAG20/#seizure-does-not-violate)
- Navigable ([Guideline 2.4](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#navigation-mechanisms))
  - [x] [Bypass Blocks](http://www.w3.org/TR/WCAG20/#navigation-mechanisms-skip)
  - [x] [Page Titled](http://www.w3.org/TR/WCAG20/#navigation-mechanisms-title)
  - [x] [Focus Order](http://www.w3.org/TR/WCAG20/#navigation-mechanisms-focus-order)
  - [x] [Link Purpose (In Context)](http://www.w3.org/TR/WCAG20/#navigation-mechanisms-refs)
- Readable ([Guideline 3.4](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#meaning))
  - [x] [Language of Page](http://www.w3.org/TR/WCAG20/#meaning-doc-lang-id)
- Predictable ([Guideline 3.2](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#consistent-behavior))
  - [x] [On Focus](http://www.w3.org/TR/WCAG20/#consistent-behavior-receive-focus)
  - [x] [On Input](http://www.w3.org/TR/WCAG20/#consistent-behavior-unpredictable-change)
- Input Assistance ([Guideline 3.3](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#minimize-error))
  - [x] [Error Identification](http://www.w3.org/TR/WCAG20/#minimize-error-identified)
  - [x] [Labels or Instructions](http://www.w3.org/TR/WCAG20/#minimize-error-cues)
- Compatible ([Guideline 4.1](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#ensure-compat))
  - [x] [Parsing](http://www.w3.org/TR/WCAG20/#ensure-compat-parses)
  - [x] [Name, Role, Value](http://www.w3.org/TR/WCAG20/#ensure-compat-rsv)

**Comment by massi94** [comment]
Ho praticamente completato queste WCAG, mancherebbero dei punti che però riguardano:
- la gestione degli errori sui form, che però ci state lavorando
- e gli ultimi due punti che sono sempre legati allo scripting

ho lasciato in sospeso anche il punto che gestisce i contenuti alternativi delle immagini, e quello della lingua, perché li controlliamo quando il sito è effettivamente tutto funzionante

**Comment by Polpetta** [comment]
@massi94 Ok! Ottimo lavoro!
